guides/products-and-variants/overview.md

```diff
@@ -0,0 +1,85 @@
+# Overview of products and variants
+
+Products and variants are integral to Solidus. While the `Spree::Product` and
+`Spree::Variant` models depend on each other, it is important to understand how
+they are different:
+
+- `Spree::Product`s track the general information about a product. This includes
+  the product description and the permalink where a customer would find the
+  product listing on a store.
+- `Spree::Variant`s track the specific information about a variant of that
+  product. The variant stores the product price, dimensions, and weight. The
+  variant provides the information required by orders and shipments.
+
+The `Spree::Variant` model provides the price for each line item in an order.
+
+The rest of this article introduces essential information for using products and
+variants in Solidus.
+
+<!-- TODO:
+  It might be worth diagramming how Spree::Products, Spree::Variants,
+  and Spree::LineItems affect an order and how it's priced.
+-->
+
+## Products
+
+`Spree::Product`s track unique products within your store. If you sell a mug and
+a t-shirt, you would set up a product for each of them.
+
+If you have a number of items that are similar (like t-shirts that come in
+small, medium, and large sizes), you can create [variants](#variants) for a
```

Comment:
This information should come first to provide context about the information listed above.

Reply:
You are very right. I've extended the products and variants bullet points here.